### PR TITLE
Skip creating temp file before calling perf_to_profile. (#277)

### PR DESCRIPTION
**Pull request by lannadorai**
Issue: #277
perf_to_profile aborts if the provided output profile *.pb.gz is already created so skip creating a temp profile file in pprof before calling perf_to_profile.

### CODE REVIEW

File: internal/driver/tempfile.go [hunk 1]
```diff
@@ -21,6 +21,19 @@ import (
 	"sync"
 )
 
+// getNextTempFile returns the next available temp file in dir with the provided prefix and suffix.
+func getNextTempFile(dir, prefix, suffix string) (string, error) {
```
Comment (aalexand) [inline internal/driver/tempfile.go]: There is quite a bit of duplicated code in here...  Can't you just use existing `newTempFile`, grab the name of the returned file using Name() and then Close() it before use?

Comment (lannadorai) [reply internal/driver/tempfile.go]: `newTempFile` creates the temp file. But this change is to skip it. I will modify `newTempFile` to use `getNextTempFile`.

**Comment by aalexand** [comment]
Did you mean to close the PR?

**Comment by lannadorai** [comment]
I wanted to send out a new patch. It didn’t appear in the PR. So, I created
a new one.

**Pull request closed without merging**